src/cunumeric.mk

```diff
@@ -15,51 +15,54 @@
+							 cunumeric/unary/unary_op.cc                                 \
+							 cunumeric/unary/unary_red.cc                                \
+							 cunumeric/unary/convert.cc                                  \
+							 cunumeric/nullary/arange.cc                                 \
+							 cunumeric/nullary/eye.cc                                    \
+							 cunumeric/nullary/fill.cc                                   \
+							 cunumeric/nullary/window.cc                                 \
+							 cunumeric/index/advanced_indexing.cc                        \
+							 cunumeric/index/choose.cc                                   \
+							 cunumeric/index/repeat.cc                                   \
+							 cunumeric/index/zip.cc                                      \
+							 cunumeric/item/read.cc                                      \
+							 cunumeric/item/write.cc                                     \
+							 cunumeric/matrix/contract.cc                                \
+							 cunumeric/matrix/diag.cc                                    \
+							 cunumeric/matrix/gemm.cc                                    \
+							 cunumeric/matrix/matmul.cc                                  \
+							 cunumeric/matrix/matvecmul.cc                               \
+							 cunumeric/matrix/dot.cc                                     \
+							 cunumeric/matrix/potrf.cc                                   \
+							 cunumeric/matrix/syrk.cc                                    \
+							 cunumeric/matrix/tile.cc                                    \
+							 cunumeric/matrix/transpose.cc                               \
+							 cunumeric/matrix/trilu.cc                                   \
+							 cunumeric/matrix/trsm.cc                                    \
+							 cunumeric/matrix/util.cc                                    \
+							 cunumeric/random/rand.cc                                    \
+							 cunumeric/random/bitgenerator.cc                            \
+							 cunumeric/random/randutil/generator_host.cc                 \
+							 cunumeric/random/randutil/generator_host_straightforward.cc \
+							 cunumeric/search/nonzero.cc                                 \
+							 cunumeric/set/unique.cc                                     \
+							 cunumeric/set/unique_reduce.cc                              \
+							 cunumeric/stat/bincount.cc                                  \
+							 cunumeric/convolution/convolve.cc                           \
+							 cunumeric/transform/flip.cc                                 \
+							 cunumeric/arg.cc                                            \
 							 cunumeric/mapper.cc
 
-GEN_CPU_SRC += cunumeric/cephes/chbevl.cc \
+GEN_CPU_SRC += cunumeric/cephes/chbevl.cc                             \
 							 cunumeric/cephes/i0.cc
 
 ifeq ($(strip $(USE_OPENMP)),1)
-GEN_CPU_SRC += cunumeric/ternary/where_omp.cc          \
+GEN_CPU_SRC += cunumeric/ternary/where_omp.cc                        \
```

Comment:
same here. (I bet this is from the IDE you're using)

Reply:
In the code, we don't use tabs; here we do. What is the tab <> #spaces equivalent I should use ?

Reply:
moved to random/random.mk to avoid such issues (and merge conflicts)